lib/middleware.ts

```diff
@@ -86,6 +103,14 @@ var defaultHandlers = [
   (req, res) => ({"status-code": res.statusCode}),
   // Ip address
   (req) => ({ip: getIp(req)}),
+
+  // Kayvee's reserved fields
+  // Log level
+  (req, res) => ({level: getLogLevel(req, res)}),
+  // Source
+  () => ({source: "kayvee"}),
```

Comment:
"source" could be among the "clever_options", we can make the library fail on init if `source` is not set.

AFAIK users shouldn't ever pass `morgan_options`, but they're left for (1) testing (2) flexibility if needed